app/views/shared/_contributors.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= return if contributors.blank? %>
+
+<% contributors.uniq.map do |contributor| %>
```

Comment:
Is `.map` doing something here I'm not understanding?

Reply:
Yeah, good call. It's probably a remnant of a previous iteration.